src/overlays/actors/ovl_En_Lift_Nuts/z_en_lift_nuts.h

```diff
@@ -8,25 +8,25 @@ struct EnLiftNuts;
 
 typedef void (*EnLiftNutsActionFunc)(struct EnLiftNuts*, PlayState*);
 
-#define ENLIFTNUTS_GET_FF00(thisx) (((thisx)->params & 0xFF00) >> 8)
+#define ENLIFTNUTS_GET_PATH_INDEX(thisx) (((thisx)->params & 0xFF00) >> 8)
 
 typedef struct EnLiftNuts {
     /* 0x000 */ Actor actor;
     /* 0x144 */ ColliderCylinder collider;
     /* 0x190 */ SkelAnime skelAnime;
     /* 0x1D4 */ EnLiftNutsActionFunc actionFunc;
-    /* 0x1D8 */ Vec3f vec_1D8;
-    /* 0x1E4 */ s32 unk_1E4;
-    /* 0x1E8 */ s32 unk_1E8;
-    /* 0x1EC */ s16* ptr_1EC;
-    /* 0x1F0 */ Vec3s jointTable[OBJECT_DNT_LIMB_MAX];
-    /* 0x298 */ Vec3s morphTable[OBJECT_DNT_LIMB_MAX];
+    /* 0x1D8 */ Vec3f waypointPos;
+    /* 0x1E4 */ s32 eyeTexIndex;
+    /* 0x1E8 */ s32 unk1E8; // Set but never used
+    /* 0x1EC */ s16* sharedMem; // Shared memory location with Actor EnGamelupy
+    /* 0x1F0 */ Vec3s jointTable[BUSINESS_SCRUB_LIMB_MAX];
+    /* 0x298 */ Vec3s morphTable[BUSINESS_SCRUB_LIMB_MAX];
     /* 0x340 */ UNK_TYPE1 unk_340[0xC];
     /* 0x34C */ s16 textId;
-    /* 0x34E */ s16 unk_34E;
+    /* 0x34E */ s16 autotalk;
     /* 0x350 */ UNK_TYPE1 unk_350[0x4];
-    /* 0x354 */ s16 unk_354;
-    /* 0x356 */ s16 unk_356;
+    /* 0x354 */ s16 frames; // Frame counter used for various different things
```

Comment:
Any reason for `frames` over `timer`?

Reply:
Mainly just because it counts up instead of down, which I thought we wanted to distinguish with a different name. I don't really have an opinion, but I'll change it to timer.

Reply:
There was a discussion about this at one point, mainly for using `timer` for counting down and `counter` for counting up, but then there was some weird cases where it could count up and down, so we leaned towards using `timer` for counting both up and down